apps/students/views.py

```diff
@@ -81,7 +81,7 @@ def attendence(request):
                     stu_attendance.save()
     else:
         context['no_class_today'] = True
-        return render(request, 'students/attendence.html', context)
+        return render(request, 'students/attendance.html', context)
```

Comment:
You need to change the name of the HTML files also present in `templates/<path-mentioned-above>` folder.